packages/nextjs/components/scaffold-eth/Navlinks.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import { BugAntIcon, SparklesIcon } from "@heroicons/react/24/outline";
+
+const NavLink = ({ href, children }: { href: string; children: React.ReactNode }) => {
+  const router = useRouter();
+
+  return (
+    <Link href={href} passHref>
+      <a
+        onClick={() => {
+          if (document.activeElement instanceof HTMLElement) {
+            document.activeElement.blur();
+          }
+        }}
```

Comment:
Also we can remove this and pass in `setIsOpen` as props to `NavLink` and use it here to close the drawer.